src/common/src/config.rs

```diff
@@ -53,15 +49,6 @@ where
     config
 }
 
-/// Map command line flag to `Flag`. Should only be used in `#[derive(OverrideConfig)]`.
-pub fn true_if_present(b: bool) -> Flag {
-    if b {
-        Some(true)
-    } else {
-        None
-    }
-}
```

Comment:
If we use default parsing function for `Option<bool>`, the expected cli would be 

<img width="804" alt="image" src="https://user-images.githubusercontent.com/52783948/222333408-582f1305-4508-4df1-81ed-871b11a6f60a.png">

whereas we simply want `--enable-jaeger-tracing`.

So I think we should bring `true_if_present` back

Reply:
nvm, I will fix it 🤣